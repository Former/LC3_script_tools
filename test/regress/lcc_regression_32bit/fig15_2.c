#include <stdio.h>

int main()
{
  int i;
  int j;

  for (i = 0; i <= 10; i++)
    j = i * 712345;
    printf("%d x 712345 = %d\n", i, j);
  return 0;
}
