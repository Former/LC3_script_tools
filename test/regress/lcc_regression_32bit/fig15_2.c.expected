11 x 712345 = 7123450
